internal/engine/local_container_build_and_deployer.go

```diff
@@ -23,7 +24,8 @@ type LocalContainerBuildAndDeployer struct {
 	k8sClient k8s.Client
 	analytics analytics.Analytics
 
-	deployInfo map[model.ManifestName]k8s.ContainerID
+	deployInfo   map[model.ManifestName]k8s.ContainerID
```

Comment:
i wonder if we should index this by image name/tag rather than manifest name

Reply:
Oooh yeah @landism this might solve the problem we were discussing about how to know when to re-fetch all this info! gonna land this as is to reduce merge conflicts but this is a neat idea for future.